nextflow_schema.json

```diff
@@ -10,8 +10,13 @@
             "type": "object",
             "fa_icon": "fas fa-terminal",
             "description": "Define where the pipeline should find input data and save output data.",
-            "required": ["input", "databases", "outdir"],
+            "required": ["outdir", "databases"],
```

Comment:
shouldn't have to be modified - `--input` should accept csv and/or PEP files (see above)